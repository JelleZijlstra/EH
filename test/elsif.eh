#!/usr/bin/ehi

f = func: args
	private argc = args.length()
	if argc > 4
		echo 'Too many arguments!'
	elsif argc < 4
		echo 'Too few arguments!'
	else
		echo 'Exactly the right number of arguments!'
	end
end

f (1, 2)
f (1, 2, 3, 4, 5)
f (1, 2, 3, 4)
