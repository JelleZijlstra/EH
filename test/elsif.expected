%%stdout%%
Too few arguments!
Too many arguments!
Exactly the right number of arguments!
%%stderr%%
